src/index.js

```diff
@@ -5,13 +5,11 @@
  * Here we should add more descriptions
  * Below are all components referenced
  *
- * - See {@link ws-header/ws-header.md|WSHeader}
- * - See {@link ws-date-picker/ws-date-picker.md|WSDatePicker}
- * - See {@link ws-dropdown/ws-dropdown.md|WSDropdown}
- * - See {@link ws-inline-edit/ws-inline-edit.md|WSInlineEdit}
- * - See {@link ws-notification/ws-notification.md|WSNotification}
- * - See {@link ws-week-picker/ws-week-picker.md|WSWeekPicker}
- * - See {@link ws-tiles-chart/ws-tiles-chart.md|WSTilesChart}
```

Comment:
Because it would generate an empty page on the styleguide which is not needed. We need the API docs for the classes and not for a simple index file